release-notes/Cloud/DCP_change_log.md

```diff
@@ -6,6 +6,22 @@ uid: DCP_change_log
 
 The dataminer.services platform gets updated continuously. This change log can help you trace when specific features and changes have become available.
 
+#### 8 November 2023 - Enhancement - CloudGateway 2.12.4, SupportAssistant 1.5.2, FieldControl 2.9.1, CoreGateway 2.13.3, ArtifactDeployer 1.5.2, Orchestrator 1.4.1 - Dependencies updated [ID_37798] [ID_37797] [ID_37784] [ID_37785] [ID_37800] [ID_37805]
+
+Several dependencies have been updated.
+
+#### 8 November 2023 - Enhancement - Orchestrator 1.4.1 - Improved logging [ID_37175]
+
+Logging for the Orchestrator DxM has been improved. In several cases, the log level has been adjusted to improve visibility of important logs.
+
+#### 8 November 2023 - Fix - ArtifactDeployer 1.5.2 & Orchestrator 1.4.1 - Issue where a timeout error was logged in ArtifactDeployer [ID_37784] [ID_37785]
```

Comment:
These RN numbers are also mentioned in the first entry above. Is this OK?

Reply:
Yes they are related